src/generatetodo.js

```diff
@@ -1,5 +1,5 @@
 function generatetodo() {
-  console.log("todo");
+  console.log("todo list");
 }
 
 export default generatetodo
```

Comment:
- [ ] You should have a list of objects in an array, with each object containing these items description (string), completed (boolean), index (number), like this:

  ```
  const tasks = [
    {
      description: 'An example description in string',
      completed: false,
      index: 1,
    },
    {
      description: 'Another example description in string',
      completed: true,
      index: 2
    }
  ];
  ```